federation-js/src/composition/DirectiveMetadata.ts

```diff
@@ -192,3 +179,23 @@ export class DirectiveMetadata {
     }
   }
 }
+
+function mergeDirectiveUsages(
+  destination: DirectiveUsages | undefined,
+  source: DirectiveUsages,
+): DirectiveUsages {
+  let directiveUsages = destination;
+  if (!directiveUsages || directiveUsages.size === 0) {
+    directiveUsages = source;
+  } else {
+    for (const [directiveName, usages] of source.entries()) {
```

Comment:
Naming: would be more clear if "usages" was "sourceUsages"?

Reply:
e6c6c12